OctoKit/Issue.swift

```diff
@@ -58,25 +104,109 @@ public extension Octokit {
             }
         }
     }
+    
+    /**
+     Fetches an issue in a repository
+     - parameter owner: The user or organization that owns the repository.
+     - parameter repository: The name of the repository.
+     - parameter number: The number of the issue.
+     - parameter completion: Callback for the outcome of the fetch.
+     */
+    public func issue(owner: String, repository: String, number: Int, completion: (response: Response<Issue>) -> Void) {
```

Comment:
I feel that integers can be limiting sometimes and it's better to use strings most of the time. We are talking about obvious mistakes here right?

```swift
client.issue("nerdishbynature", , repository: "octokit.swift", number: "2", completion: ...)
vs.
client.issue("nerdishbynature", , repository: "octokit.swift", number: "Crash when fooing bar", completion: ...)
```